### PR TITLE
Added some Styling to BETA

### PR DESCRIPTION
**Pull request by TheGreatFoxxy**
Another pull request, i know I may be keeping you busy, But this one is beta changes. Added some styling to make the Game section a bit more, "Fun". Also added a boot sound, so when the page is loaded, it plays the Windows ME boot sound.

### CODE REVIEW

**Comment by TheGreatFoxxy** [comment]
Sorry if I'm hogging your time. :)

File: beta/style/beta.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* Set Background slightly darker than white */
+#body {
+background-color: gray
+}
+
+/* Set Background color to transperent, for inheiritance reasons, trying to imitate windows app footer */
+a {
+background-color: transperent;
```
Comment (ZaDarkSide) [inline beta/style/beta.css:8]: You have a typo on this line, also please move the styling directly in index.html, there is no need for a new file and folder

File: beta/index.html [hunk 2]
```diff
@@ -1,4 +1,12 @@
 <!--suppress HtmlUnknownTarget -->
+
+<!-- Style contributed by: TheGreatFoxxy; He tried to make it look good. -->
+<style src="/style/beta.css"></style>
+
+<div id="body">
+	
+<iframe src="/sounds/boot.wav" allow="autoplay" id="audio" style="display:none"></iframe>
```
Comment (ZaDarkSide) [inline beta/index.html:8]: Probably you should use `<audio>` tag here

**Review by ZaDarkSide — CHANGES_REQUESTED** [review]

File: beta/style/beta.css (hunk 1 above)
Comment (TheGreatFoxxy) [reply beta/style/beta.css]: Gotcha, done

File: beta/index.html (hunk 2 above)
Comment (TheGreatFoxxy) [reply beta/index.html]: gotcha, done

**Comment by TheGreatFoxxy** [comment]
Hopefully these commits are worthy of your greatness! :)

**Comment by blind09** [comment]
can there also be an xp theme?

**Pull request closed without merging**